### PR TITLE
Added Octave

### PR DESCRIPTION
**Pull request by suryavamsi6**
Hello World in Octave

### CODE REVIEW

File: o/Octave.m [hunk 1]
```diff
@@ -0,0 +1 @@
+printf("Hello World!");
```
Comment (Richienb) [inline o/Octave.m:1]:
```suggestion
printf("Hello World");
```

**Review by Richienb — CHANGES_REQUESTED** [review]
Remember to update the readme.

**Comment by Richienb** [comment]
Is the file extension for Octave `.m` or `.octave`?

**Comment by suryavamsi6** [comment]
> Is the file extension for Octave `.m` or `.octave`?

it is '.m', the same as MATLAB

**Pull request merged**